addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/IPBridgeHandler.java

```diff
@@ -55,6 +55,8 @@
 
     private static final Integer SYSTEM_DBEXPORTDATETIME = 10;
 
+    private static final Integer MAX_LOGIN_ATTEMPTS = 2;
```

Comment:
Could just be int instead of Integer.  The other ones are Integer because they would end up autoboxed anyway.